docker-compose.cms.yml

```diff
@@ -5,7 +5,7 @@ services:
       # Uncomment the below line to work with local check out
       # context: ../ussf-portal-cms/
       context: https://github.com/USSF-ORBIT/ussf-portal-cms.git#main
-      dockerfile: Dockerfile
+      dockerfile: Dockerfile-gh
```

Comment:
Suggestion: Please change this one back. This is for the cms repo which doesn't have a `Dockerfile-gh` yet. But eventually we will need to change this one at that time too. 

You can see this if you use the `yarn cms:up` or `yarn portal:up` commands, which both use this file. NOTE looks like `docker-compose.client.yml` is missing the following environment var so you'll have to add it to try that command.

`NEXT_PUBLIC_PORTAL_URL=http://localhost:3000`